docs/theory/UseCases.md

```diff
@@ -42,3 +40,55 @@ the first wrap key that is in turned encrypted with the second wrap key. The
 first wrap key is stored on a removable token device, e.g. YubiKey, and the
 second wrap key is sealed in a TPM NVRAM slot. For a system to boot it must
 have launched with the correct firmware and the token must be present.
+
+## Cross-links to publications mentioning TrenchBoot
+
+### Blog posts
+
+- [Open Source DRTM with TrenchBoot for AMD processors. Introduction.](
+    https://blog.3mdeb.com/2020/2020-03-28-trenchboot-nlnet-introduction/)
+- [TrenchBoot - Open Source DRTM for AMD processors. Project's basics.](
+    https://blog.3mdeb.com/2020/2020-03-31-trenchboot-nlnet-lz/)
+- [TrenchBoot: Open Source DRTM. Landing Zone validation.](
+    https://blog.3mdeb.com/2020/2020-04-03-trenchboot-nlnet-lz-validation/)
+- [TrenchBoot: Open Source DRTM. DRTM update and meta-trenchboot implementation](
+    https://blog.3mdeb.com/2020/2020-04-30-trenchboot-nlnet-release-04/)
+- [TrenchBoot: Open Source DRTM. CI/CD system.](
+    https://blog.3mdeb.com/2020/2020-05-05-trenchboot-nlnet-ci-cd-system/)
+- [Installing TrenchBoot in UEFI environments](
+    https://blog.3mdeb.com/2020/2020-05-06-trenchboot-uefi-environment/)
+- [Starting TrenchBoot's Landing Zone from iPXE](
+    https://blog.3mdeb.com/2020/2020-06-01-ipxe_lz_support/)
+- [TrenchBoot: Open Source DRTM. GRUB's new features and TPM event log.](
+    https://blog.3mdeb.com/2020/2020-07-03-trenchboot-grub-cbfs/)
+- [DEV and IOMMU: a story of two DMA protection mechanisms](
+    https://blog.3mdeb.com/2020/2020-07-03-dev_and_iommu/)
+- [TrenchBoot: Open Source DRTM. TPM event log all the way.](
+    https://blog.3mdeb.com/2020/2020-08-13-trenchboot-event-log/)
+- [TrenchBoot: Open Source DRTM. Multiboot2 support.](
+    https://blog.3mdeb.com/2020/2020-09-07-trenchboot-multiboot2-support/)
+- [TrenchBoot: Xen hypervisor support for the TrenchBoot](
+    https://blog.3mdeb.com/2020/2020-10-15-xen-implementation-for-trenchboot/)
+- [Proof of concept implementation of RATS attestation for the TrenchBoot](
+    https://blog.3mdeb.com/2020/2020-12-14-trenchboot_attestation/)
+- [TrenchBoot Anti Evil Maid for Qubes OS](
+    https://blog.3mdeb.com/2023/2023-01-31-trenchboot-aem-for-qubesos/)
+- [TrenchBoot Anti Evil Maid - Phase 2](
+    https://blog.3mdeb.com/2023/2023-09-27-aem_phase2/)
+- [TrenchBoot Anti Evil Maid - Phase 3](
+    https://blog.3mdeb.com/2024/2024-01-12-aem_phase3/)
+- [TrenchBoot Anti Evil Maid - Phase 4](
+    https://blog.3mdeb.com/2024/2024-04-11-aem_phase4/)
+
+### Scientific Publications
+
+- [Mathias Schüpany, Martin Pirker: A Revisit of Attestable Nodes for
+    Networked Applications](https://dl.acm.org/doi/abs/10.1145/3538969.3544433)
+- [Jasmin Marmsoler, Thomas Grechenig, Florian Fankhauser: TPM 2.0 als
+    Sicherheitsmaßnahme gegen Rootkits auf Linux-basierten Desktop-Systemen](
+    https://scholar.archive.org/work/kszetgytmjhglic5cngst5dnle)
```

Comment:
Nothing wrong with different languages, but let's warn the users about it. In this case a title may be enough to guess, but in future publications it doesn't have to be.

```suggestion
    https://scholar.archive.org/work/kszetgytmjhglic5cngst5dnle) (German)
```